config/gcos.words

```diff
@@ -103,7 +103,7 @@ reserved:	COMP-15=FLOAT-LONG		# GCOS
 reserved:	COMPUTATIONAL=COMPUTATIONAL-6
 reserved:	COMPUTATIONAL-1=BINARY-SHORT	# GCOS
 reserved:	COMPUTATIONAL-2=BINARY-LONG	# GCOS
-reserved:	COMPUTATIONAL-3=COMPUTATIONAL-6	# GCOS
+reserved:	COMPUTATIONAL-3			# GCOS
```

Comment:
That won't work; you'd have to alias the long form to the shorter one, see ibm.words:

```

reserved:	COMP
reserved:	COMP-1
reserved:	COMP-2
reserved:	COMP-3
reserved:	COMP-4
reserved:	COMP-5
reserved:	COMPUTATIONAL=COMP
reserved:	COMPUTATIONAL-1=COMP-1
reserved:	COMPUTATIONAL-2=COMP-2
reserved:	COMPUTATIONAL-3=COMP-3
reserved:	COMPUTATIONAL-4=COMP-4
reserved:	COMPUTATIONAL-5=COMP-5
```